pkgs/development/libraries/SDL_audiolib/default.nix

```diff
@@ -0,0 +1,53 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, SDL2
+, pkg-config
+}:
+
+stdenv.mkDerivation rec {
+  pname = "SDL_audiolib";
+  version = "unstable-2022-04-17";
+
+  src = fetchFromGitHub {
+    owner = "realnc";
+    repo = "SDL_audiolib";
+    rev = "908214606387ef8e49aeacf89ce848fb36f694fc";
+    sha256 = "sha256-11KkwIhG1rX7yDFSj92NJRO9L2e7XZGq2gOJ54+sN/A=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+  ];
+
+  buildInputs = [
+    SDL2
+  ];
+
+  cmakeFlags = [
```

Comment:
Why are these all disabled? Is it possibly there'll be another user of SDL_audiolib that expects these?

Anyway curious if nothing else, thanks :).

Reply:
Most of them draw in other libraries and I was to lazy which of these we already have and which would need to be packaged. The game does not need these and has the options itself disabled.